src/components/Page/ProgressModalPage.stories.tsx

```diff
@@ -21,25 +21,31 @@ const Template: Story<ProgressModalPageProps> = (args) => {
 
   const clickPrev = () => {
     if (step === 1) return;
-    setStep((prev) => prev - 1);
+    setStep((prev) => Math.min(args.stepLabels.length, prev - 1));
```

Comment:
I don't know what happen but in my local the progress side is not working

Reply:
try again

Reply:
lgtm